build/configs/default_config.yaml

```diff
@@ -65,9 +65,9 @@ networks:
     ethereumNetworkId: 3
     # Latest deployed Smart Contracts for the given testnet
     contractAddresses:
-      identityFactory: "0xF86177b275E4Fcb2Ab5A3FcF5E46aab51f73D7a6"
-      anchorRepository: "0xf630d8bE5664C7FB1234CEf48b6A729cEe3B71D8"
-      invoiceUnpaid: "0x9eee8FbE3E29c96cC944F0eeC1594fe1b633dABe"
+      identityFactory: "0x0d391E72325CedC41E48053E77288a89F2E31115"
```

Comment:
Rinkeby/kovan addresses didnt change?

Reply:
Will do so in my next commit after testing locally.